sparse/_compressed/compressed.py

```diff
@@ -90,24 +90,18 @@ class GCXS(SparseArray, NDArrayOperatorsMixin):
     This is stored in GCXS format, a generalization of the GCRS/GCCS formats
     from 'Efficient storage scheme for n-dimensional sparse array: GCRS/GCCS':
     https://ieeexplore.ieee.org/document/7237032. GCXS generalizes the CRS/CCS
-    sparse matrix formats. 
-
-    The traditional CRS/CCS scheme requires (n+1) one dimensional arrays
-    and length of each one dimension array is the number of non-zero element
-    that exists in each dimension of the n-dimensional array. 
-
-    For example, to store a 4-dimensional array CRS/CCS requires 5 one dimensional
-    arrays. When the number of dimensions increases, those schemes become unusable
-    due to low data sparsity and high space and time complexity. Hence, CRS/CCS are
-    not effective enough for storing higher dimensional sparse arrays.
-
-    In GCRS/GCCS scheme, storing a sparse array of n dimensions requires only three one-dimensional
-    arrays namely which is like CRS/CCS scheme for traditional two dimensional array.
+    sparse matrix formats.
 
     For arrays with ndim == 2, GCXS is the same CSR/CSC.
     For arrays with ndim >2, any combination of axes can be compressed,
     significantly reducing storage.
 
+    Let the 3 arrays be RO, CO and VL. First element
+    of array RO is 1 and later elements are the number of
+    cumulative non-zero elements in each row for GCRS, column for
+    GCCS. CO stores column indexes of non-zero elements at each row for GCRS, column for GCCS.
+    VL stores the values of the non-zero array elements.
```

Comment:
This should be in the `CSR` docstring, then you can say `CSC transpose = CSR` and `GCXS transpose + reshape = CSR`.

Reply:
> This should be in the `CSR` docstring
CSR/CCS has another set of arrays KOi which stores the indices of the values. Whereas GCXS has no such array. I deliberately wrote that.

Reply:
It still generalizes down. ;) If you ignore the label of the array.

Reply:
Instead of simply writing CCS is the transpose of CSR, I intended to write a brief description of each kind in their respective docstrings.

Reply:
SGTM, as long as you mention that fact.

Reply:
> It still generalizes down. ;) If you ignore the label of the array.

Agreed. I'll make the changes as you mentioned.